main.py

```diff
@@ -1,7 +1,36 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        "Western",
+        "Action",
+        "Dramma"
+    ]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    actors = [
+        ["George", "Klooney"],
+        ["Kianu", "Reaves"],
+        ["Scarlett", "Keegan"],
+        ["Will", "Smith"],
+        ["Jaden", "Smith"],
+        ["Scarlett", "Johansson"]
+    ]
+
+    for actor in actors:
+        Actor.objects.create(first_name=actor[0], last_name=actor[1])
```

Comment:
You can unpuck values in for loop. 
Like` for first_name , second_name in actors:`